create_infotexts.py

```diff
@@ -303,12 +200,10 @@ def main():
     metadata_json = "SMVK-Cypern_2017-01_metadata.json"
     outpath = "./infofiles/"
 
+    # Hack to printout a wikitable to copy-paste to WikiCommons
     people = create_people_mapping_wikitable(people_mapping)
     # print(people + "\n")
 
-    places = load_places_mapping()
```

Comment:
I would still load places here since you do not want to have to reload it for each image.